mlir/lib/Dialect/Rock/Transforms/AlignTiling.cpp

```diff
@@ -362,18 +516,21 @@ static Value reconfigureLAGeneric(PatternRewriter &b,
   auto regType = laIn.getType().template cast<MemRefType>();
   auto laOut = b.create<GpuAllocOp>(loc, regType);
 
+  AffineMap outIdxMap = idxMaps.back();
```

Comment:
Random note: if  we don't have the check already, let's make sure somewhere ta the beginning that the `linalg.generic` has exactly one output.